src/state/poolsListSlice.ts

```diff
@@ -71,6 +75,16 @@ export const fetchState =
     dispatch(fetchPositions({ backd }));
   };
 
+export const fetchPreviewState = (): AppThunk => (dispatch) => {
+  const provider = new ethers.providers.InfuraProvider("kovan", INFURA_ID);
+  const backd = createBackd(provider, { chainId: 42 });
```

Comment:
Let's maybe not hard code kovan an chain id 42 here?

Reply:
My thought process here was that this preview state will only be called when the user has not connected their wallet. At that time we don't know what network they are on, and so we will have to display a network by default. I think the best network to display by default is mainnet, but we don't have contracts live there yet to display, so using kovan in the interim.

Do you think we want this preview feature to show on the site while we are still only on Kovan? Showing the Kovan pools? Or do you think it is better to delay this preview until we are live on mainnet and then show the mainnet version?

Reply:
Have changed this to use mainnet!

Also have put this feature behind a `protocolLive` check, which essentially disables it. That way we can merge this PR and just remove the flag when we go live on mainnet. Should be a bit better than keeping the PR around I think. Have added a task for mainnet launch to remove this flag and test the feature again.